src/main/java/dev/hephaestus/glowcase/client/gui/screen/ingame/HyperlinkBlockEditScreen.java

```diff
@@ -24,10 +25,17 @@ public void init() {
 
 		if (this.client == null) return;
 
-		this.urlEntryWidget = new TextFieldWidget(this.client.textRenderer, width / 10, height / 2 - 10, 8 * width / 10, 20, Text.empty());
-		this.urlEntryWidget.setText(this.hyperlinkBlockEntity.getUrl());
+		this.titleEntryWidget = new TextFieldWidget(this.client.textRenderer, width / 10, height / 2 - 30, 8 * width / 10, 20, Text.empty());
+		this.titleEntryWidget.setMaxLength(Integer.MAX_VALUE);
```

Comment:
Is this really best? I think people should have the freedom to go whatever length they'd like, but maybe we should put it at 1024 just to prevent people from accidentally pasting the entire bee movie script in there? /genq

Reply:
(I'll note that this is just a passive comment, the pr doesn't even touch this value)

Reply:
Although the text entry widget lets you paste arbitrarily large strings, the glowcase networking class already restricts it to 1024, but I think it would be a good idea to make this follow the same max length

Reply:
Should I use `URL_MAX_LENGTH` inside `GlowcaseCommonNetworking` and make it public?

Reply:
go for it! or move it to HyperlinkBlockEntity and do the same

Reply:
done up in c6f928d because i'd like to clean out our PRs today